tests/kernel/mutex/mutex_error_case/src/test_mutex_error.c

```diff
@@ -193,65 +112,106 @@ static int create_negative_test_thread(int choice)
 	return ret;
 }
 
-/* TESTPOINT: Pass a null pointer into the API k_mutex_init */
+
+/**
+ * @brief Test initializing mutex with a NULL pointer
+ *
+ * @details Pass a null pointer as a parameter then expected error happened.
```

Comment:
It better to change it like below.
"Pass a null pointer as parameter, then see if the expected error happens."

Reply:
Got it fixed, thanks!